cmd/fleet/serve.go

```diff
@@ -166,6 +188,10 @@ the way that the Fleet server works.
 				}))
 			}
 
+			if config.Logging.TracingEnabled {
+				opts = append(opts, mysql.TracingEnabled(true))
```

Comment:
Nitpick: given how it's being used you could define as `mysql.WithTracing()` (no arguments).

Reply:
I could and thought about it but for consistency with other functions in the same file I kept it taking a bool.

Reply:
Done